src/app/Shared/Services/Services.tsx

```diff
@@ -42,6 +42,11 @@ import { ApiService } from './Api.service';
 import { CommandChannel } from './CommandChannel.service';
 import { ReportService } from './Report.service';
 
+export interface Target {
```

Comment:
Should this be moved into the `Target.service.tsx`? This would be more akin to how the types are organized around `Api.service.tsx`.